### PR TITLE
examples/RCInput:add boardconfig initialization to prevent failure of…

### PR DESCRIPTION
**Pull request by dammstanger**
examples/RCInput : add boardconfig initialization to prevent failure of reading and writing channels. Has been tested in Pixhawk and Pixhawk4

### CODE REVIEW

File: libraries/AP_HAL/examples/RCInputToRCOutput/RCInputToRCOutput.cpp [hunk 1]
```diff
@@ -19,7 +29,8 @@ static uint16_t last_value[MAX_CHANNELS];
 void setup(void)
 {
     hal.console->printf("Starting RCInputToRCOutput test\n");
-
+     AP_Param::set_object_value(&BoardConfig, BoardConfig.var_info, "PWM_COUNT", FMU_PWM_COUNT);
```
Comment (khancyr) [inline libraries/AP_HAL/examples/RCInputToRCOutput/RCInputToRCOutput.cpp]: one space should be remove

File: libraries/AP_HAL/examples/RCOutput2/RCOutput.cpp [hunk 2]
```diff
@@ -77,7 +87,8 @@ MENU(menu, "Menu: ", rcoutput_menu_commands);
 
 void setup(void) {
     hal.console->printf("Starting AP_HAL::RCOutput test\n");
-
+     AP_Param::set_object_value(&BoardConfig, BoardConfig.var_info, "PWM_COUNT", FMU_PWM_COUNT);
```
Comment (khancyr) [inline libraries/AP_HAL/examples/RCOutput2/RCOutput.cpp]: one space should be remove

**Comment by dammstanger** [comment]
Thanks for reminding me the rule. I think I have found a template to follow, Randy and Peter had a commit fixing up the same bug, but only in RCInput example. So I think I can solved the rest based on their previous commit.

**Comment by dammstanger** [comment]
Randy and Peter's commit is 15ae45320578a56054e59010b858bbaaba4c8d32 10days ago.

**Pull request closed without merging**